scripts/sanity_chk/sanitylib.py

```diff
@@ -336,6 +336,7 @@ def __init__(self, instance, type_str):
         super().__init__(instance, type_str)
 
         self.terminated = False
+        self.call_west_flash = False
```

Comment:
should this be in the Handler class instead, alongside `self.call_make_run`

Reply:
Sure, will move to Handler class.

Reply:
Done